src/nat/front_ends/fastapi/response_helpers.py

```
@@ -194,3 +194,203 @@ async def generate_streaming_response_full_as_str(payload: typing.Any,
         else:
             raise ValueError("Unexpected item type in stream. Expected ChatResponseSerializable, got: " +
                              str(type(item)))
+
+
+async def generate_responses_api_streaming(
+    payload: typing.Any,
+    *,
+    session: Session,
+    model: str,
+    step_adaptor: StepAdaptor = StepAdaptor(StepAdaptorConfig()),
```

Ruff (0.14.14)

[warning] Unused function argument: `step_adaptor`

(ARG001)